docker-compose.test.yml

```diff
@@ -19,7 +11,11 @@ services:
 
   prometheus:
     volumes:
-      - ./prometheus.test.yml:/etc/prometheus/prometheus.yml 
+      - ./prometheus.test.yml:/etc/prometheus/prometheus.yml
+
+  endpoint_querier:
+    environment:
+      - LANTERN_ENDPTINFO_NETSTATS_QNAME=${LANTERN_TEST_QNAME}
```

Comment:
how do you ensure that test messages on this queue don't conflict with the capability querier test messages?

Reply:
This is a good point, it now checks to make sure the queue is empty and defers calling cleanQueue